### ISSUE TITLE
The text suggest Feature Policy is superior because sandbox whitelist approach hurt extensibility

### ISSUE THREAD

**Issue by compulim** [WICG/feature-policy#149]
In [section 3](https://wicg.github.io/feature-policy/#other-and-related-mechanisms),

> … the sandbox mechanism uses a whitelist approach which is impossible to extend without compatibility risk.
> 
> Feature Policy is intended to be used in combination with the sandbox mechanism (i.e. it does not duplicate feature controls already covered by sandbox), and provides an extensible mechanism that addresses the above limitations.

The text suggest Feature Policy address limitation on `sandbox`'s "whitelist approach". But in fact, Feature Policy is also a whitelist approach and that hurt compatibility pretty much the same way.

**Comment by clelland** [comment]
I've updated some of this text in #151, which I hope will make the distinction clearer -- essentially, we can't add a new feature to the sandbox (without significant compatibility risk with existing sites), because sandboxing automatically disables *all* features, and requires developers to update the sandbox attribute to re-allow every feature that is removed.

Feature policy at least has the option of defining features to be available in all contexts by default, while still allowing them to be disabled by developers. There could be a large number of such features added to the platform, and existing sites, sandboxed or not, should continue to work as before until a developer actually explicitly disables them in a frame.

Camera and microphone access is a different issue -- those should not be allowed in frames by default anyway (see [the HTML spec](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-allowusermedia)) There are several other features which are defined like this: allowed in the top-level document, and blocked by default in frames. In these cases, feature policy at least provides a uniform mechanism for enabling them in cross-origin frames. The number of cases where an existing established feature goes from "generally available" to "blocked in cross-origin frames" should be very small, and is mostly separate from feature policy.

**Comment by compulim** [comment]
Now I understand the bigger picture. It sounds a bit confusing because some of them are "on-by-default" (geolocation), some are "off-by-default" (camera/microphone/payment). And the control of the defaults is outside the scope of Feature Policy. Not to mention we also want to have "backward compatibility" with `sandbox` attribute.

Maybe adding some texts regarding the "on/off-by-default" and pointing to the IFRAME spec will clear some confusions.
